File: python/problem_1.py
def multiples_of_3_and_5(limit=1000):
    return sum(x if x % 3 == 0 or x % 5 == 0 else 0 for x in range(1, limit))


print(True if multiples_of_3_and_5() == 233168 else False)

File: php/problem_1.php
<?php
function multiples_of_3_and_5($limit = 1000)
{
    $ok_nums = [3, 5];

    foreach (range(6, $limit - 1) as $num) {
        if ($num % 3 == 0 or $num % 5 == 0) {
            $ok_nums[] = $num;
        }
    }

    return array_sum($ok_nums);
}

$answer = 233168;
echo multiples_of_3_and_5() == $answer ? "true" : "false";
